src/GenAPI/Microsoft.DotNet.GenAPI/SyntaxGeneratorExtensions.cs

```diff
@@ -151,14 +151,7 @@ private static SyntaxNode DestructorDeclaration(this SyntaxGenerator syntaxGener
 
     public static class SyntaxNodeExtensions
```

Comment:
Why is this class public? I don't think we want to expose it, or do we?

Reply:
Nope, we don't want to expose it. Going to make it internal